1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,234 @@
+---
+RFC:  RFCnnnn
+Author:  travisez13
+Status:  Draft
+SupercededBy:  N/A
+Version: 0.1
+Area: Engine
+Comments Due: 6/30/2019
+---
+
+# `PowerShell Core` Policy
+
+## Motivation
+
+Consumers, developers, and enterprise system administrators should be able to flexibly and reliable configure PowerShell 7.
```

Comment:
"reliably"

Reply:
```suggestion
Consumers, developers, and enterprise system administrators should be able to flexibly and reliably configure PowerShell 7.
```

Reply:
Please start using suggestion for this type of thing:
![image](https://user-images.githubusercontent.com/10873629/58736044-49ac7600-83b1-11e9-8b74-65e953c4360e.png)

Reply:
I apologize. I'm not good with GitHub, yet.